### DISCUSSION TITLE
WARNING No result data for this session available on Ergast! for all calls to get_session

### DISCUSSION THREAD

**Discussion by sltimmins** [theOehrly/Fast-F1#387] [Q&A] [answered]
Hello,

I am trying to use the package for the first time and every call to get session I get the following warnings, using v3.0.3:

`logger      WARNING 	Failed to load result data from Ergast!`
`core        WARNING 	No result data for this session available on Ergast! (This is expected for recent sessions)`

and then results in incorrect data, specifically race finishing orders as they are all NaN. Any guidance?

I am using the following code for example:
`race = fastf1.get_session(2021, 7, 'R')`
`race.load()`

**Comment by theOehrly** [comment, marked as answer]
This is caused because Ergast is having some server problems over the last few days. The likely cause is some larger server maintenance on their end. See #384

**Reply by sltimmins** [reply]
Cool thanks! Sorry I missed the other thread when I was looking through the issues.
